forestserviceprototype/specialuseform/templates/specialuseform/includes/header.html

```diff
@@ -0,0 +1,114 @@
+{% load static from staticfiles %}
+
+<header class="usa-header usa-header-extended" role="banner">
```

Comment:
is there any way to load this in from the uswds package so we can more easily update? Probably not feasible, but just checking.